### ISSUE TITLE
ImportError: cannot import name speller

### ISSUE THREAD

**Issue by DanielAndreasen** [mesbahamin/stoichiograph#5]
After installing this with `pip install stoichiograph` I also had to install `pathlib`. I believe you can include dependencies somewhere in your `setup.py` file.
Anyway, after that small fix, I get the error mentioned in the title. The source is line 9:

    from stoichiograph import speller

I think it has to be changed to

    from . import speller

Fun project! :+1:

**Comment by mesbahamin** [comment]
Thanks for letting me know about this :)

Hmm. Which version of Python are you using? I only tested this with Python 3.4, 3.5, and 3.6. I don't think `pathlib` was added to the standard library until 3.4, so that would explain why you had to install it if you're using Python <3.4.

Your Python version might also explain the other problem as well. When I do `pip install stoichiograph --user` in my home directory, or just `pip install stoichiograph` in a virtual environment, I'm able to run the `stoichiograph` command without issues.

I tend, when I can, to use absolute rather than explicit relative imports. However, some discussion on [this stackoverflow answer](https://stackoverflow.com/a/4209771) suggests that this practice is no longer as important as it once was.
